src/ActionInputValidator/Validations.ts

```diff
@@ -33,9 +33,9 @@ export function validateAppDetails() {
 
     if(!!actionParams.appName || (!!actionParams.slotName && actionParams.slotName !== 'production')) {

```

Comment:
Assuming there will not be an issue with case-sensitivity here.

Reply:
This condition is to check default case, which we ourselves have set if user doesn't provide a value